lib/git_reflow/commands/refresh.rb

```diff
@@ -13,14 +13,13 @@
   c.arg_name '[remote_location] - remote repository name to fetch updates from (origin by default),
           [base_branch] - branch that you want to merge with (master by default)'
```

Comment:
we should update this to remote the `[base_branch]` from the `arg_name` as this is now handled as a flag